lib/fluent/plugin/filter_concat.rb

```diff
@@ -55,8 +61,8 @@ def initialize
     def configure(conf)
       super
 
-      if @n_lines.nil? && @multiline_start_regexp.nil? && @multiline_end_regexp.nil? && @partial_key.nil? && !@use_partial_metadata
-        raise Fluent::ConfigError, "Either n_lines, multiline_start_regexp, multiline_end_regexp, partial_key or use_partial_metadata is required"
+      if @n_lines.nil? && @multiline_start_regexp.nil? && @multiline_end_regexp.nil? && @partial_key.nil? && !@use_partial_metadata && !@use_partial_cri_logtag
+        raise Fluent::ConfigError, "Either n_lines, multiline_start_regexp, multiline_end_regexp, partial_key, use_partial_metadata or use_partial_cri_logtag is required"
```

Comment:
It may be better to use predicate because target parameters are increased.

```
def required_params_specified?
...
end

unless required_params_specified?
   raise ...
end
```

I mean above or something more meaningful name.

Reply:
Except above point, LGTM.